screens/Budget/BudgetScreen.tsx

```diff
@@ -36,9 +42,19 @@ export default function BudgetScreen({ navigation }: RootTabScreenProps<'Budget'
         } catch (e) {
             setpasswordHash('undefined');
         }
+        if (route.params?.refresh) {
+
+        }
     }
 
-    const selectedBudget = data?.budgets.__typename == 'BudgetsSuccess' ? data.budgets.budgets.find(bud => (bud.month == month && bud.year == year)) : undefined
+    useEffect(() => {
```

Comment:
Change to use refresh. You should have it because it was merged into develop already.